yt_dlp/extractor/porndead.py

```diff
@@ -0,0 +1,127 @@
+import re
+import urllib.parse
+
+from yt_dlp.utils._utils import int_or_none
+
+from .common import InfoExtractor
+from ..utils import ExtractorError
+
+
+class PornDeadIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?porndead\.org/video/(?P<id>[0-9a-f]+)'
+    _TESTS = [
+        {
+            'url': 'https://porndead.org/video/65fefcb523810',
+            'info_dict': {
+                'id': '65fefcb523810',
+                'ext': 'mp4',
+                'title': 'Hysterical Literature - Isabel Love',
+                'age_limit': 18,
+            },
+        },
+    ]
+
+    def _real_extract(self, url):
+        url = url.strip().lower()
+
+        # if www is missing, add it because the relative URLs seem to depend on it
+        parsed = urllib.parse.urlparse(url)
+        if parsed.netloc == 'porndead.org':
+            parsed = parsed._replace(netloc='www.porndead.org')
+            url = urllib.parse.urlunparse(parsed)
+
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        # find video title ideally
+        title = (
+            self._html_search_regex(
+                r'<div[^>]+class=["\']title_video["\'][^>]*>([^<]+)</div>',
+                webpage,
+                'title',
+                default=None,
+            )
+            or self._og_search_title(webpage, default=None)
+            or f'Video {video_id}'
+        )
+
+        # extract variable player_url from <script> player_url = "..." </script>
+        player_rel = self._search_regex(
+            r'(?is)player[_-]?url\s*=\s*(["\'])(?P<u>[^"\']+)\1',
+            webpage,
+            'player url',
+            default=None,
+            group='u',
+        )
+
+        if not player_rel:
+            raise ExtractorError('Could not find player_url on page', expected=True)
+
+        # resolve relative URL and append type=1 like the JS on the page does
+        player_url = urllib.parse.urljoin(url, player_rel)
+        player_endpoint = player_url + ('&type=1' if '?' in player_url else '?type=1')
+
+        ajax_headers = {
+            'Referer': url,
+            'X-Requested-With': 'XMLHttpRequest',
+            'User-Agent': 'Mozilla/5.0 (compatible)',
+            'Accept': '*/*',
+        }
+
+        # get the options html
+        options_html = None
+        try:
+            options_html = self._download_webpage(
+                player_endpoint,
+                video_id,
+                headers=ajax_headers,
+                data=b'',  # empty body to force POST where supported
+            )
+        except Exception as e:
+            raise ExtractorError(
+                f'Failed to download options from {player_endpoint}: {e}',
+                expected=True,
+            )
```

Comment:
`options_html` is also required for extraction, so make this fatal:
 ```suggestion
        options_html = self._download_webpage(
            player_endpoint,
            video_id,
            headers=ajax_headers,
            data=b'',  # empty body to force POST where supported
        )
```
Sending an empty data string is the usual way to force a `POST`.